lib/Db/Folder.php

```diff
@@ -30,8 +30,8 @@ class Folder extends Entity implements IAPI, \JsonSerializable
     protected $opened = true;
     /** @var int|null */
     protected $deletedAt = 0;
-    /** @var int|null */
-    protected $lastModified = 0;
+    /** @var string|null */
+    protected $lastModified = '';
```

Comment:
The value is basically always set so the default doesn't matter as much, but what about going for '0'